storm-client/src/jvm/org/apache/storm/stats/StatsUtil.java

```diff
@@ -1589,18 +1570,16 @@ public static ComponentPageInfo aggCompExecsStats(
             if (lastReportedTime != null) {
                 reportedTime = lastReportedTime;
             } else {
-                reportedTime = 0;
+                reportedTime = lastReportedTime = 0;
             }
         }
 
         if (lastNimbusTime == null || !reportedTime.equals(lastReportedTime)) {
             lastNimbusTime = Time.currentTimeSecs();
         }
 
-        ret.put("is-timed-out", Time.deltaSecs(lastNimbusTime) >= timeout);
```

Comment:
Is it safe to remove two key-values? Just to check again.

Reply:
yes, it's safe we will refresh the is-timed-out when we make assignments